golden-ami/build/bin/network-validator.go

```
@@ -0,0 +1,199 @@
+	httpClient.Transport = &http.Transport{
+		Proxy:           http.ProxyFromEnvironment,
+		TLSClientConfig: tlsClientConfig,
+	}
+
+	fmt.Printf("Validating %s\n", endpoint)
+
+	// Retry up to maxRetries times
+	for i := 0; i < *maxRetries; i++ {
+		switch port {
+		case 80:
+			_, err = httpClient.Get(fmt.Sprintf("%s://%s", "http", host))
```

golangci-lint (2.12.2)

[error] response body must be closed

(bodyclose)